lib/Service/BoardService.php

```diff
@@ -662,6 +672,16 @@ public function clone($id, $userId) {
 		]);
```

Comment:
We can separate the board cloning logics in a different class - The BoardMapper or the Service.